pennylane/plugins/default_qubit.py

```diff
@@ -85,8 +89,9 @@ def __init__(self, wires, *, shots=1000, analytic=True):
         self.eng = None
         self.analytic = analytic
 
-        self._state = np.zeros(2 ** wires, dtype=complex)
+        self._state = np.zeros(2 ** wires, dtype=np.complex128)
```

Comment:
Was this also changed due to performance?

Reply:
I actually don't remember, it's actually the same thing.